openedx/core/djangoapps/api_admin/management/commands/create_api_access_request.py

```diff
@@ -0,0 +1,130 @@
+""" Management command to create an ApiAccessRequest for given users """
+import logging
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
+
+from openedx.core.djangoapps.api_admin.models import (
+    ApiAccessConfig,
+    ApiAccessRequest,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Create an ApiAccessRequest for the given user
+
+    Example usage:
+        $ ./manage.py lms create_api_request <username> --create-config
+    """
+
+    help = 'Create an ApiAccessRequest for the given user'
+    DEFAULT_STATUS = ApiAccessRequest.APPROVED
+    DEFAULT_WEBSITE = 'www.test-edx-example-website.edu'
+    DEFAULT_REASON = 'Generated by management job create_api_request'
+
+    def add_arguments(self, parser):
+        parser.add_argument('username', type=str)
+        parser.add_argument(
```

Comment:
Why do we need this as an argument? Shouldn't this be the default behavior?

Reply:
I didn't want to assume. I don't totally understand the use of this field, so when I added optional args to pass in all other fields, I though I may as well do this as well.

Reply:
Well, I rather not complicate the management command API. Let's remove the argument now and we can also add it when we truly need it.